lib/capsize/deployer.rb

```diff
@@ -123,6 +123,28 @@ def self.cvs_root_defintion?(val)
       val.index(':') == 0 && val.scan(":").size > 1
     end
 
+    def print_parameter(parameter)
+      name = parameter.name
+      val = parameter.value
+
+      val = deployment.prompt_config[name] if parameter.prompt?
+
+      return nil if val.nil?
+      val.strip!
+      return set_string_parameter(name, val) if val =~ /\A\d+\D+/
+
+      case val[0]
+      when ":", "%", "[", "{", /\d/
```

Comment:
`\d` and the rest of the when statement is checking the first character of `val`, so `foo2` would be a string parameter as well as `2foo` which would be caught by the above guard clause.